docs/DeveloperGuide.md

```diff
@@ -540,40 +540,63 @@ testers are expected to do more *exploratory* testing.
 
    1. Re-launch the app by double-clicking the jar file.<br>
-       Expected: The most recent window size and location is retained.
+      Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+1. Shutdown
+
+   1. Shut down the app using the `exit` command.<br>
+      Expected: The app shuts down.
 
 ### Deleting a client
 
 1. Deleting a client while all clients are being shown
 
-   1. Prerequisites: List all clients using the `list` command. Multiple clients in the list.
+   1. Prerequisites: List all clients using the `listclients` command. Multiple clients in the list.
 
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `deleteclient 1`<br>
+      Expected: First client is deleted from the list. Details of the deleted client shown in the status message.
 
-   1. Test case: `delete 0`<br>
+   1. Test case: `deleteclient 0`<br>
       Expected: No client is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect delete client commands to try: `deleteclient`, `deleteclient x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+1. Deleting a client while no clients are being shown
+
+   1. Prerequisite: Use `findclients keyword` command (where keyword matches none of the clients) to display an empty list of clients.<br>
+
+   1. Test case: `deleteclient 1`<br>
+      Expected: No client is deleted. Error details shown in the status message. Status bar remains the same.
+
+   1. Other delete client commands to try: `deleteclient`, `deleteclient x`, `...` (where x is anything)<br>
+      Expected: Similar to previous. Note that there is no valid command because the client list is empty.
 
 ### Saving data
 
-1. Dealing with missing/corrupted data files
+1. Dealing with missing data files
 
-   1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+   1. Prerequisite: The folder containing the jar file contains a `data` folder.
 
-1. _{ more test cases …​ }_
+   1. Delete the `data` folder then launch the app by double-clicking the jar file.<br>
+      Expected: Shows the GUI with a set of clients, tasks and orders. The window size may not be optimum.
+
+1. Dealing with corrupted data files
+
+   1. Prerequisite: The folder containing the jar file contains a `data` folder, which contains `addressbook.json`, `salesBook.json` and `taskBook.json`. 
+
+   1. Test case: Corrupt the `addressbook.json` file by deleting a closing curly bracket (`}`) or any other brackets.<br>
+      Expected: Shows the GUI with no clients, but with a previously saved set of tasks and orders.
```

Comment:
This is not true because there is a possibility that the orders/tasks are wiped too if the customer of an order does not exist in the new 'empty' client list. 
If I remember correctly, there are two possibilities for this case (all data is wiped (if any order exists) or only previous tasks persist (if all the tasks are tagged with `General`))

Reply:
Hmm seems like the behavior changed. I will update it. Thanks for pointing out.

Reply:
I tried, even if all tasks are tagged with `General`, they will also be wiped out.

Reply:
Correction:
If I remember correctly, there are two possibilities for this case (all data is wiped (if any order exists) or only previous tasks persist (if all the tasks are tagged with General and the order list is empty))

Reply:
This is actually an implementation bug. It should behave like what I stated.

Reply:
You may want to wait until #294 to be merged so that you can observe the exact behaviour.

Reply:
Okay, I will update it now.